R/deploy.R

```diff
@@ -120,8 +120,8 @@ bundle_dir <- function(path = ".", filename = fs::file_temp(pattern = "bundle",
   setwd(path)
   on.exit(expr = setwd(before_wd), add = TRUE)
 
-  message(glue::glue("Bundling directory {path}"))
-  check_bundle_contents(path)
+  message(glue::glue("Bundling current directory ({path})"))
```

Comment:
Perhaps we should move this up above `setwd` (and leave the wording as "Bundling directory"). I think "Bundling current directory" would be confusing to a user receiving the message if they are not bundling their current directory